src/ui/rcb-ui.svelte

```diff
@@ -66,13 +67,17 @@
 
   onMount(() => {
     if (!isInElement) {
-      document.body.style.overflow = "hidden"; // Prevents background scrolling
+      document.documentElement.style.height = "100%";
```

Comment:
trick to prevent ever having two scrolls (once you finish scrolling on the modal), if the caller page has to scroll also, goes along with overflow: hidden